core/vdbe/mod.rs

```diff
@@ -55,25 +56,6 @@ pub type CursorID = usize;
 
 pub type PageIdx = usize;
 
-#[allow(dead_code)]
-#[derive(Debug)]
```

Comment:
oh yeah was curious about these... LSP didn't show they were in use as all the match statements were referencing the `function::Func` type and not this one. These all set to remove?

Reply:
Yeah, looks like it's dead code. For future: please resist the urge to include (random) cleanups in your pull requests. It's better to do them separately because (1) review is easier and (2) reduces risk of merge conflicts.